contracts/HydraDelegation/DelegationPoolLib.sol

```diff
@@ -21,9 +22,10 @@ library DelegationPoolLib {
      * @param pool the DelegationPool for rewards to be distributed to
      * @param amount the total amount to be distributed
      */
-    function distributeReward(DelegationPool storage pool, address staker, uint256 amount) internal {
+    function distributeReward(DelegationPool storage pool, uint256 amount) internal {
         if (amount == 0) return;
-        if (pool.virtualSupply == 0) revert NoTokensDelegated(staker);
+        // In our version of the contracts we cannot hit this case, because virtualSupply will always equal the supply
```

Comment:
And what if supply is zero?

Reply:
It should just leave the function, otherwise the calculation will fail